frontend/controller/serviceworkers/cache.js

```diff
@@ -103,23 +104,14 @@ if (
         return cache
           .match(request, { ignoreSearch: true, ignoreVary: true })
           .then((cachedResponse) => {
-            if (request instanceof URL) {
-              console.error('@@@@XX', 106, request)
-            }
             if (cachedResponse) {
               // If we're offline, return the cached response, if it exists
               if (navigator.onLine === false) {
-                if (request instanceof URL) {
-                  console.error('@@@@XX', 112, request, navigator.onLine)
-                }
                 return cachedResponse
               }
             }
 
-            return fetch(request.clone?.() || request).then(async (response) => {
-              if (request instanceof URL) {
-                console.error('@@@@XX', 120, request, response.status)
-              }
+            return fetch(event.request.clone?.() || request).then(async (response) => {
```

Comment:
This is what seems to have fixed the issue. Note that I'm still using the original request here instead of the 'rewritten'  `request`.